Documentation/DataSources/PrivateWorkbooksDeprecation.md

```diff
@@ -71,4 +71,8 @@ Learn more about Workbook Access control here: https://docs.microsoft.com/en-us/
 
 # What happens if I don't Convert? 
 
-If you do not convert your private workbook to shared it will not be accessible in Azure Portal after **June 30, 2021**. 
\ No newline at end of file
+If you do not convert your private workbook to shared it will not be accessible in Azure Portal after **June 30, 2021**. 
```

Comment:
i'd change this to "not directly accessible _from the Workbooks tools in the Azure Portal_ after July 2021"